test/load/test_load_tx_ethereum.py

```diff
@@ -0,0 +1,268 @@
+import time
+import threading
+import copy 
+
+import siftool_path
+from siftool import eth, test_utils, sifchain
+from siftool.eth import NULL_ADDRESS
+from siftool.common import *
+from web3.eth import Contract
+import web3
+# In separate window: test/integration/framework/siftool run-env
+# test/integration/framework/venv/bin/python3 test/load/test_load_tx_ethereum.py
+
+fund_amount_eth = 10 * eth.ETH
+fund_amount_sif = 10 * test_utils.sifnode_funds_for_transfer_peggy1
+rowan_contract_address = "0x5FbDB2315678afecb367f032d93F642f64180aa3"
+
+threads_num = 3
+amount_to_send = 10000
+
+def bridge_bank_lock_burn(eth_tx_wrapper: eth.EthereumTxWrapper, bridge_bank_sc: Contract, test_eth_account: str, 
+    recipient: str, amount_to_send: int, nonce: int, token_sc: Contract = None, isLock: bool = True):
```

Comment:
There's a function-scoped variable here called `amount_to_send`, whose name is the same as a defined module variable in L19 above. This may lead to confusion.

Reply:
will rename the variable.